libs/payments/customer/src/lib/invoice.manager.ts

```diff
@@ -78,7 +78,22 @@ export class InvoiceManager {
       requestObject
     );
 
-    return stripeInvoiceToFirstInvoicePreviewDTO(upcomingInvoice);
+    let last4: string | undefined;
```

Comment:
Ah, thank you! You are correct - an upcoming invoice has none of this information, this is leftover from early debugging, but because of the `undefined` case handling, it didn't impact the functionality of the method. Will remove.